openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -90,7 +77,7 @@ def test_handle(self, commit, mock_task):
         call_command('backpopulate_program_credentials', commit=commit)
 
         if commit:
-            mock_task.assert_called_once_with(self.alice.username)
+            mock_task.assert_called_with(self.alice.username)
```

Comment:
Got it. If you go with my suggestion above (to check `is_learner_issuance_enabled` before scheduling tasks), you can probably change this back, too.